server/src/main/java/keywhiz/service/daos/AclDAO.java

```diff
@@ -52,14 +53,17 @@
   private GroupDAO groupDAO;
   private SecretContentDAO secretContentDAO;
   private SecretSeriesDAO secretSeriesDAO;
+  private ObjectMapper mapper;
```

Comment:
Not sure what the JVM infers. Causes a compilation error if you forget to initialize it and makes it clearer to readers which fields are expected to be modifiable.